rpc/lib/Simple/RPC/Client.hs

```diff
@@ -182,15 +228,27 @@ runAtAs :: Executable -> SSHConfig -> Username -> Runner IntermediateRep
 runAtAs exe sshConf uname = with (exec exe & onSSH sshConf & asUser uname)
 
 -- NOTE: This should be removed from here. Leave the installation to the user.
+-- XXX We can avoid sudo access or have a separate API which does not require
+-- sudo access.
+
+-- XXX scpExecutable
+
+-- | @installOnRemote localPath sshEndpoint remotePath@. Install the RPC
+-- endpoint server executable on a remote machine using @scp@. Requires @sudo@
+-- access to create the directory if it does not exist.
```

Comment:
This is for backward compatibility.
To be removed.
Installing the rpc executable should be the burden of the user not the library.